packages/nextjs/components/burnerwallet/QrCodeReader.tsx

```diff
@@ -54,6 +55,9 @@ const QrCodeReader = () => {
             </div>
           ) : (
             <div className="max-w-[90%] w-[300px] h-[300px] fixed top-0 left-0 right-0 bottom-0 m-auto z-[100]">
+              <button className="pointer-events-auto" onClick={() => setIsQrReaderOpen(false)}>
+                <XCircleIcon className="w-6 h-6" />
```

Comment:
Is not showing to me in light mode, should we force the font color to white?

```suggestion
                <XCircleIcon className="w-6 h-6 text-white" />
```

Reply:
Good call, got this fixed and aligned the button to the right